packages/broker/src/ConfigWizard.ts

```diff
@@ -225,7 +234,7 @@ export const selectDestinationPathPrompt = {
     }
 }
 
-const selectValidDestinationPath = async (): Promise<inquirer.Answers> => {
+const selectStoragePath = async (): Promise<inquirer.Answers> => {
     const answers = await inquirer.prompt([selectDestinationPathPrompt])
```

Comment:
`generateOrImportPrivateKey` is used for the `when` methods in inquirer, which we don't test for. The validation function takes the imported private key from the `input` field rather than from the `answers`, hence the passing tests.

This aside the renamings seem alright

Reply:
We could remove all which don't effect tests (e.g. `generateOrImportPrivateKey` and `revealGeneratedPrivateKey` in `should exercise the happy path with user input`)